### ISSUE TITLE
Consider renaming `Export Wallet…` and `Import Wallet…`

### ISSUE THREAD

**Issue by casey** [sparrowwallet/sparrow#1302]
I was looking for label import and export, and I originally couldn't find it because I didn't think to look in `Import Wallet…` and `Export Wallet…`, since those sounded like they were for importing / exporting the whole wallet, i.e., including private keys.

I was thinking that maybe they should be named `Import…` and `Export…` since they're general, and support a bunch of things other than importing/exporting the whole wallet.

**Comment by craigraw** [comment]
Yes, I agree the menu item name isn't a great fit when it comes to labels. But just using `Import...` and `Export...` might be net loss since at least all the functions are wallet related. People might start thinking this is the place "import" a transaction they want to broadcast, for example. I don't like it, but maybe `Import Wallet or Labels...`

**Comment by raphjaph** [comment]
You could also consider moving the export/import of labels into the `Tools` section. That's where I went to look for it first until I found this issue.
